Tracker/Stats/StatsViewController.swift

```diff
@@ -21,24 +27,83 @@ final class StatsViewController: UIViewController {
     private lazy var image: UIImageView = {
         let image = UIImageView()
         image.image = UIImage(named: "smile")
+        image.isHidden = true
         image.translatesAutoresizingMaskIntoConstraints = false
         return image
     }()
     
     private lazy var emptyLabel: UILabel = {
         let label = UILabel()
         label.text = "Анализировать пока нечего"
-        label.textColor = .black
+        label.textColor = .textColor
         label.font = .systemFont(ofSize: .init(18), weight: .medium)
+        label.isHidden = true
         label.translatesAutoresizingMaskIntoConstraints = false
         return label
     }()
     
+    private lazy var card1: StatCardView = {
```

Comment:
Для улучшения читаемости лучше называть карточки по их предназначению
```suggestion
    private lazy var perfectStreakCard: StatCardView = {
```